Packs/Palo_Alto_Networks_WildFire/Integrations/WildFireReports/WildFireReports.py

```diff
@@ -21,9 +21,10 @@ def get_agent():
         within XSOAR (both on-prem and cloud).
         """
         platform = get_demisto_version().get('platform')
-        if platform == 'x2':
-            return 'xsoartim' if is_demisto_version_ge('8.0.0') else 'xdr'
-        return 'xsoartim'
+        if platform == 'x2':  # XSIAM
+            return 'xdr'
+        else:  # XSOAR (on-prem or cloud)
+            return 'xsoartim'
```

Comment:
simpler not mandatory tho
```suggestion
        return 'xdr' if platform == 'x2' else 'xsoartim':  # XSIAM
```

Reply:
Added, thanks.